api/proto/ank_base.proto

```diff
@@ -201,7 +201,17 @@ message AgentMap {
 /**
 * A message that has not yet been implemented but will contain attributes of the agent in the future.
 */
-message AgentAttributes {}
+message AgentLoad {
+    uint32 cpu_usage = 1;
+    uint64 free_memory = 2;
+}
+
+/**
+* A message that contains attributes of the agent.
+*/
+message AgentAttributes {
+    AgentLoad agent_resources = 1;
+}
```

Comment:
We can use separate objects for the two values to take care of the optional requirements of the interface and also solve some problems with the conversions and string representation:
```suggestion
// TODO
message CpuLoad {
    uint32 cpu_load = 1; // TODO
}

// TODO
message FreeMemory {
        uint64 free_memory = 1; // TODO
}

/**
* A message that contains attributes of the agent.
*/
message AgentAttributes {
    CpuLoad cpu_load = 1; // TODO
    FreeMemory free_memory = 2; // TODO
}
```

In YAML:
```yaml
agents:
  agent_A:
    cpu_load: 130
    free_memory: 243234234
```

Reply:
Added the corresponding serialization and enabled filtering. Please confirm changes.